.changeset/odd-squids-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Update R2 CreateBucket action to include the storage class in the request body.
```

Comment:
please use the recommended message format as per https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets